forms/checkValidation.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "checkValidation",
+    "class": "checkValidation-form",
+    "fields": [
+        {
+            "name": "gender",
+            "label": "female",
+            "type": "radio",
+            "value": "female",
+            "validation": "required",
+            "class": "txt-fullname",
+            "wrapperClass": "txt-fullname-wrapper"
+        },
+        {
+            "name": "gender",
+            "label": "male",
+            "type": "radio",
+            "value": "male",
+            "validation": "required",
+            "class": "txt-fullname",
+            "wrapperClass": "txt-fullname-wrapper"
+        },
+        {
+            "name": "number",
+            "label": "Number Only",
+            "type": "text",
+            "placeholder": "Number Only",
+            "validation": "required|phone-lk",
+            "class": "txt-numbers",
+            "wrapperClass": "txt-number-wrapper"
+        },
+        {
+            "name": "date",
+            "label": "Date of Birth",
+            "type": "date",
+            "placeholder": "Date of Birth",
+            "validation": "required|date",
+            "class": "txt-address",
+            "wrapperClass": "date-dob-wrapper"
+        },
+        {
+            "name": "email",
+            "label": "EMAIL",
+            "type": "email",
+            "placeholder": "Email",
+            "validation": "required|email",
+            "class": "txt-email",
+            "wrapperClass": "txt-nic-email"
+        },
+        {
+            "name": "phone",
+            "label": "phone number",
+            "type": "number",
+            "placeholder": "Phone Number",
+            "validation": "required|phone-lk|",
```

Comment:
@hariidev should not have extra `|` at the end... I'm not sure if that will cause a problem?